src/ExternalTrackerGoogleAnalyticsBundle/Bridge/Chameleon/ExternalTracker/ExternalTrackerGoogleUniversalAnalytics.php

```diff
@@ -10,6 +10,9 @@
 use TPkgExternalTrackerState;
 use TShopBasketArticle;
 
+/**
+ * @deprecated Google Universal Analytics will be removed in July 2023
```

Comment:
This could hint on which class replaces it.

Reply:
:heavy_check_mark: Added a hint, that `ExternalTrackerGoogleAnalyticsGa4` is a replacement.